meta-neutis-bsp/recipes-bsp/u-boot/u-boot_2018.07.bb

```diff
@@ -50,7 +50,7 @@ do_configure_append() {
 
 EXTRA_OEMAKE_append_sun50i = " BL31=${DEPLOY_DIR_IMAGE}/bl31.bin "
 
-do_compile_sun50i[depends] += "atf-sunxi:do_deploy"
```

Comment:
I think it is not good idea to use do_compile depend for all sunxi families, only sun50i is depended on ATF.